pkg/testgridanalysis/testgridanalysisapi/types.go

```diff
@@ -82,5 +81,5 @@ const (
 
 var (
 	OperatorInstallPrefix          = "operator install "
-	OperatorConditionsTestCaseName = regexp.MustCompile("operator (install|conditions) (?P<operator>.*)")
+	OperatorConditionsTestCaseName = regexp.MustCompile("operator install (?P<operator>.*)")
```

Comment:
we really can't get this test renamed to better align with what it is actually reporting on?

Reply:
> we really can't get this test renamed to better align with what it is actually reporting on?

There will still be an install test.  it will simply change to be virtual.  But you need to track the old value or you won't be able to read your historical data.  This code handles both properly so when https://github.com/openshift/release/pull/12298 merges, we won't have trouble.